core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -440,6 +442,10 @@ oppia.directive('testInteractionPanel', [
               answer: answer
             });
           };
+
+          $scope.submitAnswerFromButton = function() {
```

Comment:
Probably better to rename to onSubmitAnswerFromButton.

Just to check, though, why was this changed? (On the surface it seems unrelated to this PR; was it an existing bug?)

Reply:
Yep, it's an existing bug. There was no submit button. So pressing `enter` was `submit` but that does not work for complex interactions like code repl where `enter` is a newline. So I added a submit button.

Reply:
Actually, you would find a few more things changed / removed / added. The original thing was all broken up.

Reply:
Ah, got it, thanks -- I think I missed that this was in testInteractionPanel, I thought it was something showing up in the UI already.